src/Signature/SignatureV4.php

```diff
@@ -82,6 +82,7 @@ public function presign(
         $expires,
         array $options = []
     ) {
+        $expiresTimestamp = $this->convertToTimestamp($expires);
```

Comment:
@daum Thanks for working through this with us. I think we're both on the same page with regards to the problem and that the solution being worked towards both fixes the issues at hand and maintains our contracts.

For a relative expires, which will be the only used case for an optional `$relativeTimeBase ` in `convertToTimestamp`, we'll already have `$startTimestamp` and the intent of the contract is the relative expires is based on that. The default/non-existent value should just be null, as that will have `strtotime` use its own default if we're not overriding. This should probably also warrant a new test on the relative time handling of SignatureV4 of the '+6 days' the suite is set up for; you'll have to pass in the 'start_time' based on the 'aws_time' as an option.